packages/u/unity/xmake.lua

```diff
@@ -0,0 +1,24 @@
+package("unity")
+    set_homepage("http://www.throwtheswitch.org/unity")
+    set_description("Simple Unit Testing for C")
+    set_license("MIT")
+
+    add_urls("https://github.com/ThrowTheSwitch/Unity/archive/refs/tags/$(version).tar.gz", "https://github.com/ThrowTheSwitch/Unity")
```

Comment:
add a newline and use git url `https://github.com/ThrowTheSwitch/Unity.git`

```lua
add_urls("https://github.com/ThrowTheSwitch/Unity/archive/refs/tags/$(version).tar.gz", 
         "https://github.com/ThrowTheSwitch/Unity.git")
```

Reply:
OK